Modules/SubModule_Import/xASL_imp_DCM2NII_ConvertScan.m

```diff
@@ -88,7 +88,8 @@
             bPutInSessionFolder = false;
     end
 
-    if ~isempty(strfind(char(imPar.folderHierarchy(end)),'PAR'))
+    % Check for old PARREC data
```

Comment:
I just removed this in #938

Reply:
Okay? Then it will be a part of the conflict resolving, I guess?

Reply:
Yes, that's going to be part of resolving indeed. BUT, in #938, Henk did an equivalent fix as you did here and moved the code elsewhere. It is thus easier to restore this (with a new commit) to the previous state - it will save us some hassle with the conflict solving.

Reply:
3e79423 ?

Reply:
Reverted 👍